puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/rubygemRpmNightlyRelease.groovy

```diff
@@ -0,0 +1,31 @@
+pipeline {
```

Comment:
A nit, to capitalize RPM in the file name since its an acronym.

Reply:
renamed to `rubygemRPMNightlyRelease.groovy`.